enforcer-rules/pom.xml

```diff
@@ -8,10 +8,16 @@
     </parent>
     <artifactId>incrementals-enforcer-rules</artifactId>
     <dependencies>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-core</artifactId>
+            <scope>provided</scope>
+        </dependency>
         <dependency>
             <groupId>org.apache.maven.enforcer</groupId>
-            <artifactId>enforcer-rules</artifactId>
-            <version>3.0.0-M1</version>
+            <artifactId>enforcer-api</artifactId>
+            <version>3.2.1</version>
+            <scope>provided</scope>
```

Comment:
Depending on the public API as recommended upstream rather than on internal implementation details.